dm/syncer/ddl.go

```diff
@@ -396,6 +398,18 @@ func (ddl *DDLWorker) HandleQueryEvent(ev *replication.QueryEvent, ec eventConte
 	return ddl.strategy.handleDDL(qec)
 }
 
+func (ddl *DDLWorker) handleSkippedDDL(ec *eventContext) error {
+	// skipped ddl includes:
+	// - ddls that dm don't handle, such as analyze table(can be parsed), create function(cannot be parsed)
+	// - ddls related to db/table which is filtered
+	// for those ddls we still flush job like non-skipped ddls, so we can set global checkpoint.
+	// then checkpoint can match master position if skipped ddl is the last binlog in source db
+	if err := ddl.flushJobs(); err != nil {
```

Comment:
I'm a bit concerned that it will cause DML replication concurrency drop, because we will let block-list tables' DDL clear DML queue. Can we only record the location, and wait for a heartbeat event (idle for 30s) to update the sync state?

Reply:
not safe to update global point before dml before it is synced if worker failover in the middle

Reply:
https://github.com/pingcap/tiflow/pull/8193/commits/1a6eb7300041e02457febd475c836a69ffa1f9e9